hugo/content/8_release-updates/2023.md

```diff
@@ -7,6 +7,19 @@ weight: 802
 
 This section of the document gives you an overview of the new features and functions in rapyuta.io. It includes new features, improvements, announcements, deprecations, and bug fixes.
 
+
+## April 19
+
+| Feature | Type | Description |
+| ------- | ---- | ----------- |
+| Dectl Config command | New | You can now use the config command to set custom configurations on the device. |
+| [Folder Node](/3_how-tos/32_device-management/324_applying-configuration-on-devices/) | New | You can now create multiple folders and sub-folders to organize the configuration parameters.| 
+| Package Version  | Improvement | You now have the option to switch the package versions if a package has more than one version.|
```

Comment:
**suggestion**: `You now have the option to switch the package version from the package details page, if a package has more than one version`.